.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+- '6'
+- '5'
+- '4'
```

Comment:
For projects with an `.nvmrc`, we can leave this blank and have the `.nvmrc` dictate the node version to test.